phpunit.xml.dist

```diff
@@ -22,7 +28,6 @@
 
     <php>
         <ini name="precision" value="8" />
-        <env name="KERNEL_CLASS" value="Sonata\DoctrinePHPCRAdminBundle\Tests\Fixtures\App\Kernel" />
```

Comment:
Either this env var is defined in custom bootstrap somehow , added to dev-kit or this file cannot be generic

Reply:
I tried 
```
public static function getKernelClass(): string
    {
        return Kernel::class;
    }
```

Seems like it doesn't work...

Reply:
@sonata-project/contributors Any idea ?